tests/common/dualtor/mux_simulator_control.py

```diff
@@ -167,7 +167,7 @@ def _post(server_url, data):
         server_url = '{}?reqId={}'.format(server_url, uuid.uuid4())  # Add query string param reqId for debugging
         logger.debug('POST {} with {}'.format(server_url, data))     # lgtm [py/clear-text-logging-sensitive-data]
         headers = {'Accept': 'application/json', 'Content-Type': 'application/json'}
-        resp = requests.post(server_url, json=data, headers=headers)
+        resp = requests.post(server_url, json=data, headers=headers, timeout=600)
```

Comment:
This is just to send a HTTP request. Timeout value do not need to be as much as 600 seconds. How about just 5 seconds?

Reply:
Per discussion, I change it to 10s.